From a191425bdac5171d1abd4b7318dc977dd381bb7b Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 26 Sep 2020 16:06:57 +0200
Subject: [PATCH] default to <4GB image base for now

It can still be enabled via --default-image-base-high
---
 ld/emultempl/pep.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ld/emultempl/pep.em b/ld/emultempl/pep.em
index 0918a68059..ff47f05186 100644
--- a/ld/emultempl/pep.em
+++ b/ld/emultempl/pep.em
@@ -159,7 +159,7 @@ static int support_old_code = 0;
 static lang_assignment_statement_type *image_base_statement = 0;
 static unsigned short pe_dll_characteristics = DEFAULT_DLL_CHARACTERISTICS;
 static bfd_boolean insert_timestamp = TRUE;
-static bfd_boolean high_default_bases = TRUE;
+static bfd_boolean high_default_bases = FALSE;
 static const char *emit_build_id;
 
 #ifdef DLL_SUPPORT
-- 
2.28.0.windows.1

